infrastructure/terraform/variables.tf

```
@@ -4,7 +4,7 @@
 
 variable "prefix" {
   type    = string
-  default = "catenacax1"
+  default = "catenaxprs"
```

```suggestion
  default = "catenacax2"
```
should not be prs-specific by default